tagstudio/src/qt/ts_qt.py

```diff
@@ -27,21 +27,22 @@
 from PySide6.QtWidgets import (QApplication, QWidget, QHBoxLayout, QPushButton, QLineEdit, QScrollArea, QFileDialog,
 							   QSplashScreen, QMenu)
 from humanfriendly import format_timespan
-
-from src.core.library import ItemType
-from src.core.ts_core import (PLAINTEXT_TYPES, TagStudioCore, TAG_COLORS, DATE_FIELDS, TEXT_FIELDS, BOX_FIELDS, ALL_FILE_TYPES,
+from tagstudio.src.core.library import Collation, Entry, ItemType, Library, Tag
+from tagstudio.src.core.palette import ColorType, get_tag_color
+from tagstudio.src.core.ts_core import TagStudioCore
+from tagstudio.src.core.constants import (PLAINTEXT_TYPES, TAG_COLORS, DATE_FIELDS, TEXT_FIELDS, BOX_FIELDS, ALL_FILE_TYPES,
```

Comment:
`Tagstudio.src...` import format breaks at runtime for vscode and when running from the terminal

```suggestion
from src.core.library import Collation, Entry, ItemType, Library, Tag
from src.core.palette import ColorType, get_tag_color
from src.core.ts_core import TagStudioCore
from src.core.constants import (PLAINTEXT_TYPES, TAG_COLORS, DATE_FIELDS, TEXT_FIELDS, BOX_FIELDS, ALL_FILE_TYPES,
```